contracts/MakerOrders.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.4.18;
+
+
+import "./Withdrawable.sol";
+
+
+contract MakerOrders is Withdrawable {
+    struct FreeOrders {
+        uint32 firstOrderId;
+        uint32 numOrders; //max is 256
+        uint8 maxOrdersReached;
+        uint256 takenBitmap;
+    }
+
+    //each maker will have orders that will be reused.
+    mapping(address => FreeOrders) public makerOrdersSell;
+    mapping(address => FreeOrders) public makerOrdersBuy;
+
+    function takeOrderId(FreeOrders storage freeOrders)
+        internal
+        onlyAdmin
```

Comment:
since internal and extended. don't need only Admin no more...

Reply:
Done.